<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessShow</class>
 <widget class="QWidget" name="ChessShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>331</width>
     <height>451</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>70</y>
     <width>82</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>双人对战</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>ai对战</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>网络对战</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>返回登录</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_select" native="true">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>191</width>
     <height>141</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_ai_first">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ai先手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_aibehind">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>ai后手</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_NetWar" native="true">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>300</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>82</width>
      <height>76</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>放弃寻找</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_findteam">
       <property name="text">
        <string>寻找队伍</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>680</y>
     <width>24</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>状态:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_stable">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>680</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>240</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>悔棋</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>710</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>710</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>鼠标点击的点</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_show">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>290</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="inter_face_seting" native="true">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>410</y>
     <width>161</width>
     <height>221</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>101</width>
      <height>62</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QPushButton" name="pushButton_set_back_ground">
       <property name="text">
        <string>设置棋盘背景</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_setprebackground">
       <property name="text">
        <string>设置前景</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="pushButton_10">
       <property name="text">
        <string>先手棋子设置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_12">
       <property name="text">
        <string>后手棋子设置</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="chess_widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_37">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix4_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix3_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/2.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix2_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/3.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix2_7">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/7.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix5_5">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/6.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_38">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_39">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_40">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_41">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_42">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="chess_widget" native="true">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>541</width>
     <height>121</height>
    </rect>
   </property>
   <widget class="QPushButton" name="pushButton_36">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>关闭</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix4_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix3_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/2.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix2_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/3.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix2_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/7.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_back_pix5_4">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>55</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../build-NewGoBang-unknown-Debug/background/6.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_31">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_32">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_33">
    <property name="geometry">
     <rect>
      <x>184</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_34">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_35">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
